ci/ray_ci/docker_container.py

```diff
@@ -51,8 +52,10 @@ def _get_image_version_tags(self) -> List[str]:
         branch = os.environ.get("BUILDKITE_BRANCH")
         sha_tag = os.environ["BUILDKITE_COMMIT"][:6]
         pr = os.environ.get("BUILDKITE_PULL_REQUEST", "false")
+
         if branch == "master":
-            return [sha_tag, "nightly"]
+            formatted_date = datetime.now().strftime("%y%m%d")
+            return [sha_tag, f"nightly.{formatted_date}"]
```

Comment:
Do we need to update documentation (https://docs.ray.io/en/latest/ray-overview/installation.html#docker-source-images) and inform user of the nightly images?

Reply:
@khluu , we should tag both "nightly" and "nightly.yymmdd" at the same time.

we should stop tagging the `sha_tag` for containers. for that, we should update the doc first (remote the SHA prefix row and related texts), and maybe post a notice to the community slack channel somewhere, and then we remove the `sha_tag` here.

Reply:
Removing support for SHA tag on doc here: https://github.com/ray-project/ray/pull/43719

Reply:
@aslonnie That essentially means `postmerge` would no longer push tags to Docker Hub anymore except for nightly runs right?

Reply:
yes, that is right for master branch. only nightly runs on postmerge on master branch push to dockerhub

we still need to / want to keep pushing the dockerhub for release branches.

Reply:
got it

Reply:
this is fixed